### PR TITLE
LXGW WenKai TC: Version 1.330;April 28, 2024 added



### PR DESCRIPTION
**Pull request by aaronbell**
Taken from the upstream repo https://github.com/aaronbell/LxgwWenkaiTC at commit https://github.com/aaronbell/LxgwWenkaiTC/commit/a5cf76f5bd1f26cdf74ca1b22b9f6fe17b8d5bc5.
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
- [ ] Minisite_url definition in the METADATA.pb file for commissioned projects
- [ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [ ] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

**Comment by emmamarichal** [comment]
### **Outlines / Spacing review:**

Check all the masters! The errors are the same in light, regular and bold.

- Found some spacing issues. Here is the list of glyphs that have more than -20 on right or left:
`Aogonek/Eogonek/Ohorn/Uhorn/u0181/u0187/u018A/u0191/u0193/u0198/u019D/u01A4/u01B3/u024A/u1E28/u1EDA/u1EDC/u1EDE/u1EE0/u1EE2/u1EE8/u1EEA/u1EEC/u1EEE/u1EF0/uniA7AA/aogonek/hcircumflex/u0180/u0188/u01B4/u01E9/u021F/u0221/u0235/u024B/u0256/u0257/u025D/u0260/u0272/u0273/u027B/u0290/u029B/u02A0/u02AE/u02AF/u1EE9/u1EEB/u1EED/u1EEF/u1EF1/uhorn/uogonek/u0496/u0497/Epsilontonos/Etatonos/Iotatonos/Omicrontonos/Upsilontonos/Omegatonos/u03D3/u1F08/u1F09/u1F0A/u1F0B/u1F0C/u1F0D/u1F0E/u1F0F/u1F88/u1F89/u1F8A/u1F8B/u1F8C/u1F8D/u1F8E/u1F8F/u1F18/u1F19/u1F1A/u1F1B/u1F1C/u1F1D/u1FC8/u1FC9/u1F28/u1F29/u1F2A/u1F2B/u1F2C/u1F2D/u1F2E/u1F2F/u1FCA/u1FCB/u1F98/u1F99/u1F9A/u1F9B/u1F9C/u1F9D/u1F9E/u1F9F/u1F38/u1F39/u1F3A/u1F3B/u1F3C/u1F3D/u1F3E/u1F3F/u1FDA/u1FDB/u1F48/u1F49/u1F4A/u1F4B/u1F4C/u1F4D/u1FF8/u1FF9/u1FEC/u1F59/u1F5B/u1F5D/u1F5F/u1FEA/u1FEB/u1F68/u1F69/u1F6A/u1F6B/u1F6C/u1F6D/u1F6E/u1F6F/u1FFA/u1FFB/u1FA8/u1FA9/u1FAA/u1FAB/u1FAC/u1FAD/u1FAE/u1FAF/u31B7/u3099/u309A/u0308/u0307/gravecomb/acutecomb/u030B/u0302/u030C/u0306/u030A/tildecomb/u0304/u0305/hookabovecomb/u030D/u030F/u0311/u0316/u0317/u0318/u0319/u031A/u031B/u031C/u031D/u031E/u031F/u0320/dotbelowcomb/u0324/u0325/u0326/u0327/u0328/u0329/u032A/u032C/u032D/u032E/u032F/u0330/u0331/u0332/u0334/u0335/u0336/u0338/u0339/u033A/u033B/u033C/u033D/u0340/u0341/u0358/u0361/u02BE/glyph33712/glyph33714/glyph33720/glyph33722/glyph33724/glyph33726/uFFFFD/acutecomb.1/uni0308.1/uni2015/uni3016/uni3017/uniFF3F/uniFFE3`

Screenshot 2024-05-16 at 16 22 53

- caron in dcaron looks a bit tight:
Screenshot 2024-05-16 at 16 11 56

- cedilla placement
Screenshot 2024-05-16 at 16 15 50

File: ofl/lxgwwenkaitc/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<p>This project is the Traditional Chinese version of "霞鶩文楷". Initially, it mainly utilized AFDKO in conjunction with the legacy glyph shapes provided by the Zonz community to convert the characters contained in Klee One into old character forms, and supplemented the old version of "霞鶩文楷" with modifications. Some components and characters were further manually modified. Subsequently, reference was made to the "Legacy Glyph Standardization Document" from Yidianzifang to modify most components, making it more suitable for Traditional Chinese users and enthusiasts of legacy glyph forms.</p>
```
Comment (emmamarichal) [inline ofl/lxgwwenkaitc/DESCRIPTION.en_us.html:1]: @aaronbell, copy/paste the translation of the repo description here. Feel free to correct it directly here.

**Comment by emmamarichal** [comment]
We still have an issue with the family name:

Screenshot 2024-05-16 at 16 47 13


🔥 FAIL Check family name for GF Guide compliance.
"霞鶩文楷等寬 TC" contains the following characters which are not allowed: "寬文楷等霞鶩".
"霞鶩文楷等寬 TC" doesn't start with an uppercase letter.

**Comment by aaronbell** [comment]
> We still have an issue with the family name:
> 
> Screenshot 2024-05-16 at 16 47 13
> 🔥 FAIL Check family name for GF Guide compliance. "霞鶩文楷等寬 TC" contains the following characters which are not allowed: "寬文楷等霞鶩". "霞鶩文楷等寬 TC" doesn't start with an uppercase letter.

Aren’t these the localized names? Do you want them stripped out entirely?

**Comment by emmamarichal** [comment]
I just discussed about it with the team, we can let it like this. It's an issue from fontbakery! Let's try to onboard it like this :)

File: ofl/lxgwwenkaitc/DESCRIPTION.en_us.html (hunk 1 above)
Comment (aaronbell) [reply ofl/lxgwwenkaitc/DESCRIPTION.en_us.html]: Seemed fine to me! I just removed one word for better flow. Thanks

**Comment by aaronbell** [comment]
> * Aogonek/Eogonek/Ohorn/Uhorn/u0181/u0187/u018A/u0191/u0193/u0198/u019D/u01A4/u01B3/u024A/u1E28/u1EDA/u1EDC/u1EDE/u1EE0/u1EE2/u1EE8/u1EEA/u1EEC/u1EEE/u1EF0/uniA7AA/aogonek/hcircumflex/u0180/u0188/u01B4/u01E9/u021F/u0221/u0235/u024B/u0256/u0257/u025D/u0260/u0272/u0273/u027B/u0290/u029B/u02A0/u02AE/u02AF/u1EE9/u1EEB/u1EED/u1EEF/u1EF1/uhorn/uogonek/u0496/u0497/Epsilontonos/Etatonos/Iotatonos/Omicrontonos/Upsilontonos/Omegatonos/u03D3/u1F08/u1F09/u1F0A/u1F0B/u1F0C/u1F0D/u1F0E/u1F0F/u1F88/u1F89/u1F8A/u1F8B/u1F8C/u1F8D/u1F8E/u1F8F/u1F18/u1F19/u1F1A/u1F1B/u1F1C/u1F1D/u1FC8/u1FC9/u1F28/u1F29/u1F2A/u1F2B/u1F2C/u1F2D/u1F2E/u1F2F/u1FCA/u1FCB/u1F98/u1F99/u1F9A/u1F9B/u1F9C/u1F9D/u1F9E/u1F9F/u1F38/u1F39/u1F3A/u1F3B/u1F3C/u1F3D/u1F3E/u1F3F/u1FDA/u1FDB/u1F48/u1F49/u1F4A/u1F4B/u1F4C/u1F4D/u1FF8/u1FF9/u1FEC/u1F59/u1F5B/u1F5D/u1F5F/u1FEA/u1FEB/u1F68/u1F69/u1F6A/u1F6B/u1F6C/u1F6D/u1F6E/u1F6F/u1FFA/u1FFB/u1FA8/u1FA9/u1FAA/u1FAB/u1FAC/u1FAD/u1FAE/u1FAF/u31B7/u3099/u309A/u0308/u0307/gravecomb/acutecomb/u030B/u0302/u030C/u0306/u030A/tildecomb/u0304/u0305/hookabovecomb/u030D/u030F/u0311/u0316/u0317/u0318/u0319/u031A/u031B/u031C/u031D/u031E/u031F/u0320/dotbelowcomb/u0324/u0325/u0326/u0327/u0328/u0329/u032A/u032C/u032D/u032E/u032F/u0330/u0331/u0332/u0334/u0335/u0336/u0338/u0339/u033A/u033B/u033C/u033D/u0340/u0341/u0358/u0361/u02BE/glyph33712/glyph33714/glyph33720/glyph33722/glyph33724/glyph33726/uFFFFD/acutecomb.1/uni0308.1/uni2015/uni3016/uni3017/uniFF3F/uniFFE3

It looks like this is for the Mono version? If you are happy enough with the performance of this non-Mono version, I say let's move it forward and then can address these Mono issues later. @chrissimpkins Unless Translate is planning on using the Mono version right away?

File: ofl/lxgwwenkaitc/DESCRIPTION.en_us.html (hunk 1 above)
Comment (NightFurySL2001) [reply ofl/lxgwwenkaitc/DESCRIPTION.en_us.html]: > "Legacy Glyph Standardization Document" from Yidianzifang

The official document name is > "Inherited Glyph Standardization Documents" from I. Font Project <.
Quoted from https://github.com/ichitenfont/inheritedglyphs/blob/master/Readme_eng.md.

**Review by emmamarichal — APPROVED** [review]

**Pull request merged**